cmd/di.go

```diff
@@ -438,8 +438,8 @@ func (di *Dependencies) bootstrapNetworkComponents(options node.OptionsNetwork)
 	}
 
 	//override defined values one by one from options
-	if options.DiscoveryAPIAddress != metadata.DefaultNetwork.DiscoveryAPIAddress {
-		network.DiscoveryAPIAddress = options.DiscoveryAPIAddress
+	if options.APIAddress != metadata.DefaultNetwork.APIAddress {
```

Comment:
What kind of API is it? Further, what kind of address is it? If you name it `MysteriumApiUrl`, then all ambiguities are gone

Reply:
Comment addressed